addons/mail/static/src/core/common/navigable_list.js

```diff
@@ -19,7 +19,6 @@ export class NavigableList extends Component {
         onSelect: { type: Function },
         options: { type: Array },
         optionTemplate: { type: String, optional: true },
-        placeholder: { type: String, optional: true },
```

Comment:
I removed the support of `placeholder` because all provided values were the same `Loading`.
There's no desire for not showing it, and I wanted to add ellipsis so it looks nicer with spinner (`Loading…`), so I took the opportunity to kill the placeholder to keep change DRY.